eslint.config.js

```diff
@@ -88,6 +88,7 @@ export default [
       'sonarjs/mouse-events-a11y': 'off', // this is also checked by the jsx-a11y
       'sonarjs/no-unstable-nested-components': 'warn',
       'sonarjs/slow-regex': 'error',
+      'sonarjs/no-misused-promises': 'off',
```

Comment:
Are there issues with these eslint rules or why are all these three now turned off? If there is some reasoning, it might be good to add it to PR's description.